frontend/src/toolbar/index.tsx

```diff
@@ -1,14 +1,75 @@
 import '~/styles'
 import './styles.scss'
 
+import { KeaPlugin, resetContext } from 'kea'
+import { formsPlugin } from 'kea-forms'
+import { loadersPlugin } from 'kea-loaders'
+import { localStoragePlugin } from 'kea-localstorage'
+import { routerPlugin } from 'kea-router'
+import { subscriptionsPlugin } from 'kea-subscriptions'
+import { waitForPlugin } from 'kea-waitfor'
+import { windowValuesPlugin } from 'kea-window-values'
 import type { PostHog } from 'posthog-js'
 import { createRoot } from 'react-dom/client'
 
-import { initKea } from '~/initKea'
 import { ToolbarApp } from '~/toolbar/ToolbarApp'
 import { ToolbarParams } from '~/types'
-;(window as any)['ph_load_toolbar'] = function (toolbarParams: ToolbarParams, posthog: PostHog) {
-    initKea()
+
+interface InitKeaProps {
+    state?: Record<string, any>
+    routerHistory?: any
+    routerLocation?: any
+    beforePlugins?: KeaPlugin[]
+}
+
+const initKeaInToolbar = ({ routerHistory, routerLocation, beforePlugins }: InitKeaProps = {}): void => {
```

Comment:
we were using the init kea from the main app but it has error handling specific to how the app operates, let's just have a separate one for the toolbar